src/index.js

```diff
@@ -1,10 +1,15 @@
 // @flow
 import p from 'path'
 import * as t from 'babel-types'
+import murmur from 'murmurhash3js'
```

Comment:
Why is this library?

Reply:
What: an efficient hashing library to create unique but short hashes

Why: wanted to use the defaultMessage as part of the ID but they are difficult to sanitize and might be quite long. This is the same hash used in the Babel react intl hash plugin

Reply:
👍